src/ui/components/PasswordInput/PasswordInput.tsx

```diff
@@ -10,73 +10,94 @@ import "./passwordInput.scss";
+              className="password-input__info-tooltip"
+              title={
+                <div>
+                  <p>Password requirements:</p>
 
-                <p>- At least 8 characters</p>
+                  <p>- At least 8 characters</p>
 
-                <p>- At least 1 upper case and letter</p>
+                  <p>- At least 1 upper case and letter</p>
 
-                <p>- At least 1 lower case letter</p>
+                  <p>- At least 1 lower case letter</p>
 
-                <p>- At least 1 special character (!@#$%^&*)</p>
+                  <p>- At least 1 special character (!@#$%^&*)</p>
 
-                <p>- At least 1 number</p>
-              </div>
-            }
-          >
-            <Icon className="password-input__info-icon" fontAwesome="fa-info" />
-          </Tooltip>
-        ) : null}
+                  <p>- At least 1 number</p>
+                </div>
+              }
+            >
+              <Icon className="password-input__info-icon" fontAwesome="fa-info" />
+            </Tooltip>
+          ) : null}
 
-        {isShowPassword ? (
-          <Tooltip
-            key={2}
-            className="eye-tooltip"
-            data-testid="eye-slash-button"
-            title="Hide Password"
-            onClick={onShowPassword}
-          >
-            <Icon className="password-input__info-icon" fontAwesome="fa-eye-slash" />
-          </Tooltip>
-        ) : (
-          <Tooltip
-            key={2}
-            className="eye-tooltip"
-            data-testid="eye-look-button"
-            title="Show Password"
-            onClick={onShowPassword}
-          >
-            <Icon className="password-input__info-icon" fontAwesome="fa-eye" />
-          </Tooltip>
-        )}
-      </InputAdornment>
-    }
-    errorMessage={errorMessage}
-    id="password"
-    inputRef={inputRef}
-    label="Password"
-    type={isShowPassword ? "text" : "password"}
-    {...passwordInputProps}
-  />
+          {isShowPassword ? (
+            <Tooltip
+              key={2}
+              className="eye-tooltip"
+              data-testid="eye-slash-button"
+              title="Hide Password"
+              onClick={onShowPassword}
+            >
+              <Icon className="password-input__info-icon" fontAwesome="fa-eye-slash" />
+            </Tooltip>
+          ) : (
+            <Tooltip
+              key={2}
+              className="eye-tooltip"
+              data-testid="eye-look-button"
+              title="Show Password"
+              onClick={onShowPassword}
+            >
+              <Icon className="password-input__info-icon" fontAwesome="fa-eye" />
+            </Tooltip>
+          )}
+        </InputAdornment>
+      }
+      errorMessage={errorMessages[0]}
+      id="password"
+      inputRef={ref}
+      label="Password"
+      type={isShowPassword ? "text" : "password"}
+      {...Object.values(passwordInputProps)[0]}
+    />
+
+    {isShowConfirmPassword ? (
```

Comment:
PasswordInput must have only one `Input`component.
You need to use it for onboarding like this 

```
<PasswordInput
     isShowHint
     id="password"
     label="Password"
     errorMessage={errors.password}
     isShowPassword={isShowPassword}
     onShowPassword={onShowPassword}
     {...register("password")}
/>

<PasswordInput
     id="confirmPassword"
     label="Confirm password"
     errorMessage={errors.confirmPassword}
     isShowPassword={isShowPassword}
     onShowPassword={onShowPassword}
     {...register("confirmPassword")}
/>      

```